app/Common/Enums/ErrorCodes.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Common\Enums;
+
+/**
```

Comment:
Here we need to use our standard exceptions. I will add those exceptions for you.